sync/Dockerfile

```diff
@@ -9,12 +9,12 @@ ARG PIP_NO_CACHE_DIR=off \
     PIP_DISABLE_PIP_VERSION_CHECK=on
 
 # Copy files and install requirements
-WORKDIR /app
+WORKDIR /sync
 COPY requirements.txt .
 RUN pip install -r ./requirements.txt
 
 # Start the app
-WORKDIR /app/src
```

Comment:
Generally, please stick with /app inside containers for simplicity and consistency.